math/finding_number_of_digits_in_a_number.cpp

```diff
@@ -40,6 +42,28 @@ uint64_t finding_number_of_digits_in_a_number(uint64_t n) {
     return count;
 }
 
+/**
+ * @brief This function finds the number of digits
+ * in constant time
+ * TC: O(1)
+ * @param n the number to check its digits
+ * @returns the digits count
+ */
+uint64_t count_digits(uint64_t n) {
+    // to handle the negative numbers
+    if (n < 0) {
+        n = -n;
+    }
+    // log(0) is undefined
+    else if (n == 0) {
+        // handling the value 0
+        return 0;
+    }
+    uint64_t count = floor(log10(n) + 1);
+
+    return count;
```

Comment:
```suggestion
    return floor(log10(n) + 1); // NOLINT(cppcoreguidelines-narrowing-conversions)
```

Though I would say this is not a good practice, and I would rather you make the function return type `count_digits` to be `double` instead. But I leave that up to @Panquesito7 to decide 😄

Reply:
I've changed the return type to avoid clang-tidy warnings